pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -87,6 +87,11 @@ func init() {
 	cloudprovider.RegisterCloudProvider(ProviderName, func(config io.Reader) (cloudprovider.Interface, error) { return newGCECloud(config) })
 }
 
+// Raw access to the underlying GCE service, probably should only be used for e2e tests
```

Comment:
So I'm a bit confused by your follow-up comments. Do you think this is fine as-is? Or should we instead export a `ListForwardingRules()` function?